archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,166 @@
+#else
+#set ($eeArtifactId = "jakarta.jakartaee-api")
+#end
+#if (${jakartaVersion} == '10')
+#set ($eeApiVersion = "10.0.0")
+#elseif (${jakartaVersion} == '9.1')
+#set ($eeApiVersion = "9.1.0")
+#elseif (${jakartaVersion} == '9')
+#set ($eeApiVersion = "9.0.0")
+#else
+#set ($eeApiVersion = "8.0.0")
+#end
+#if (${jakartaVersion} == '8')
+#set ($tomeeVersion = "8.0.13")
+#set ($payaraVersion = "5.2022.5")
+#set ($glassfishContainerId = "glassfish5x")
+#if (${profile} == 'web')
+#set ($glassfishUrl = "https://repo.maven.apache.org/maven2/org/glassfish/main/distributions/web/5.1.0/web-5.1.0.zip")
+#else
+#set ($glassfishUrl = "https://repo.maven.apache.org/maven2/org/glassfish/main/distributions/glassfish/5.1.0/glassfish-5.1.0.zip")
+#end
+#else
+#set ($payaraVersion = "6.2022.2")
+#set ($tomeeVersion = "9.0.0.RC1")
+#set ($glassfishContainerId = "glassfish7x")
+#if (${profile} == 'web')
+#set ($glassfishUrl = "https://repo.maven.apache.org/maven2/org/glassfish/main/distributions/web/7.0.0/web-7.0.0.zip")
+#else
+#set ($glassfishUrl = "https://repo.maven.apache.org/maven2/org/glassfish/main/distributions/glassfish/7.0.0/glassfish-7.0.0.zip")
+#end
+#end
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>${groupId}</groupId>
+	<artifactId>${artifactId}</artifactId>
+	<version>${version}</version>
+	<packaging>war</packaging>
+
+	<name>jakartaee-hello-world</name>
+	<description>
+  	    This is a very simple Jakarta EE application generated by the official Eclipse Starter.
+    </description>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.report.sourceEncoding>UTF-8</project.report.sourceEncoding>
+		#if (${javaVersion} == '8')
+		<maven.compiler.target>1.8</maven.compiler.target>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		#else
+		<maven.compiler.release>${javaVersion}</maven.compiler.release>
+		#end
+		<jakartaee-api.version>${eeApiVersion}</jakartaee-api.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>jakarta.platform</groupId>
+			<artifactId>${eeArtifactId}</artifactId>
+			<version>${jakartaee-api.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		#if ((${runtime} == 'payara') and (${profile} != 'full'))
+		<!-- Embedded Jakarta EE runtime -->
+		<dependency>
+			<groupId>fish.payara.extras</groupId>
+			<artifactId>payara-micro</artifactId>
+			<version>${payaraVersion}</version>
+			<scope>provided</scope>
+		</dependency>		
+		#end
+	</dependencies>
+
+	<build>
+		<finalName>jakartaee-hello-world</finalName>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.8.0</version>
```

Comment:
Lets put all versions in the <properties> section for easy of maintenance

Reply:
It's addressed. Feel free to mark resolved?